TEMPLATES/.gitleaks.toml

```diff
@@ -1,21 +1,11432 @@
+#---------------------------------------------------------------------------#
+# Authors: André Breuer, Pierre Gronau                                      #
+# Created: 2022-03-22                                                       #
+# Updated: 2024-01-18														#
+# This configuration is created from the default config template available  #
+# at:                                                                       #
+# https://github.com/zricethezav/gitleaks/blob/master/config/gitleaks.toml  #
+#                                                                           #
+# Source:                                                                   #
+#  https://gitlab.com/ndaal_open_source/ndaal_secretes_search/-/raw/master/gitleaks_base.toml #
+#                                                                           #
+# License: MIT                                                              #
+# Copyright (C) 2023, 2024 ndaal GmbH & Co KG                               #
+#---------------------------------------------------------------------------#
 
-title = "gitleaks config"
+title = "ndaal gitleaks config"
 
-[extend]
-# useDefault will extend the base configuration with the default gitleaks config:
-# https://github.com/zricethezav/gitleaks/blob/master/config/gitleaks.toml
-useDefault = true
+#---------------------------------------------------------------------------#
+# Default rules                                                             #
+#---------------------------------------------------------------------------#
+
+[[rules]]
+# See: https://about.gitlab.com/releases/2021/11/22/gitlab-14-5-released/#new-gitlab-access-token-prefix-and-detection
+id = "gitlab-pat"
+description = "GitLab Personal Access Token"
+tags = ["gitlab", "pat", "access token"]
+regex = '''glpat-[0-9a-zA-Z\-\_]{20}'''
+
+[[rules]]
+description = "GitLab Pipeline Trigger Token"
+id = "gitlab-ptt"
+regex = '''glptt-[0-9a-f]{40}'''
+tags = ["gitlab", "pipeline", "token"]
+
+[[rules]]
+description = "GitLab Runner Registration Token"
+id = "gitlab-rrt"
+regex = '''GR1348941[0-9a-zA-Z\-\_]{20}'''
+tags = ["gitlab", "runner", "token"]
+
+[[rules]]
+id = "aws-access-token"
+description = "AWS"
+tags = ["aws", "key", "token"]
+regex = '''((A3T[A-Z0-9]|AKIA|AGPA|AIDA|AROA|AIPA|ANPA|ANVA|ASIA)[A-Z0-9]{16})'''
+secretGroup = 1
+
+# Cryptographic keys
+[[rules]]
```

Comment:
Why does GitHub show me this in red? Are they some special characters ?

Reply:
@echoix It's shown in red, since there is no empty line between the comment `# Cryptographic keys` and the `[[rules]]`